tests/requirements.in

```diff
@@ -8,4 +8,5 @@ jsonschema
 pylint
 semantic_version
 pydantic
-tenacity
\ No newline at end of file
+tenacity
```

Comment:
Include only test related packages which are used currently.

Reply:
Also Dockerfile.tests should be modified to install pip packages from tests/requirements.txt.

Reply:
Missed out tenacity somehow,removing it now.

Reply:
I see that quite few unused packages in both requirments.in and tests/requirements.in. You can use `pipreqs` to find only the used package in the source.
 `pip install pipreqs`

Reply:
@arajkumar I used pipreqs find packages.

Reply:
And I only included them in the requirements.in and then created requirements.txt with pip-compile.